### PR TITLE
3.5.3 changelog

### PR DESCRIPTION
**Pull request by Bob-Murphy**
Changelog for 3.5.3 release

### CODE REVIEW

**Review by jaj22 — APPROVED** [review]

File: changelog.rst [hunk 1]
```diff
@@ -3,6 +3,43 @@ Changelog
...
+
+.. admonition:: Bugfixes
+
+   * :issue:`3195` Fixed a start-up hang when mods with too many cosmetic items were loaded
+   * :issue:`3196` Fixed incorrect order of strings for fast travel parameters
+   * :issue:`3196` Fixed an exploit in the fast travel function
+   * :issue:`3197` Fixed an issue where bipods would sometimes be deleted by the arsenal
+   * :issue:`3207` Fixed a script error with CBA when spawning AI units
+   * :issue:`3212` Fixed a script error from a debug command in the Garage
+   * :issue:`3215` Player corpses should now not be deleted until 15 minutes have passed, fixing an issue where many respawning players would delete corpses.
```
Comment (Tiny-DM) [inline changelog.rst:32]: period at the end of the line here, inconsistent with the rest. your decision if you want to change. not big though

File: changelog.rst [hunk 2]
```diff
@@ -3,6 +3,43 @@ Changelog
...
 
+Version 3.5.3
+===============
+
+.. note::
+   14th April 2024
+
+.. admonition:: Minor
+
+   * :issue:`3188` Removed a case-sensitive check from fn_initZones and implemented toLower
```
Comment (Tiny-DM) [inline changelog.rst:14]: Might want to be a bit more clear on this, it's a bit hard to understand right now

**Review by Tiny-DM — APPROVED** [review]
Two minor issues, otherwise looks good.

File: changelog.rst (hunk 1 above)
Comment (Bob-Murphy) [reply changelog.rst]: updated

File: changelog.rst (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.rst]: updated

**Review by Tiny-DM — APPROVED** [review]

**Review by ante185 — APPROVED** [review]
lgtm

**Pull request merged**